A51-custom-backend-metrics.md

```diff
@@ -306,38 +316,38 @@ data using the exposed APIs as follows. All the methods are thread safe.
 ```java
 public class OrcaServiceImpl implements OpenRcaService {
 
-// Allow configuring minimum report interval. If not configured or badly configured non-positive, 
-// the default is 30s.
-// In the future, it might be useful to let users specify minimum report interval and default 
-// report interval separately. If the default report interval is not configured or badly configured 
-// non-positive, the default is 1 min. 
-// public OpenRcaServiceImpl(long minInterval, TimeUnit timeUnit, long 
-// defaultInterval, TimeUnit)
-public OrcaServiceImpl(long minReportInterval, TimeUnit unit);
+  // Allow configuring minimum report interval. If not configured or badly configured non-positive, 
+  // the default is 30s.
+  // In the future, it might be useful to let users specify minimum report interval and default 
+  // report interval separately. If the default report interval is not configured or badly configured 
+  // non-positive, the default is 1 min. 
```

Comment:
Is this 1 minute relating to the API that doesn't exist? It is strange having the API commented out as part of a different constructor. If you want to document the potential future API, just make it separate and give it its own documentation, but then comment it out.